packages/models-library/src/models_library/api_schemas_directorv2/clusters.py

```diff
@@ -95,10 +98,13 @@ class ClusterGet(Cluster):
         alias="accessRights", default_factory=dict
     )
 
+    # TODO[pydantic]: The `Config` class inherits from another class, please create the `model_config` manually.
+    # Check https://docs.pydantic.dev/dev-v2/migration/#changes-to-config for more information.
     class Config(Cluster.Config):
         allow_population_by_field_name = True
 
-    @root_validator(pre=True)
+    @model_validator(mode="before")
+    @classmethod
```

Comment:
double classmethod

Reply:
Yes, a bug of the bump-pydantic tool creates double @classmethod, I'm working also on them.

See: https://github.com/pydantic/bump-pydantic/pull/160